### DISCUSSION TITLE
Analytical defocusing correction for pole figure in mtex

### DISCUSSION THREAD

**Discussion by nhgoo** [mtex-toolbox/mtex#1983] [Ask Anything]
Is there any body to know the analytical defocusing correction method in mtex ? 
I mean mtex function to do it. 

I found the defocusing data set in the mtex example data library. the file name is "ZnCuTi_defocusing_PF_002_R.uxd". 
What is the difference between "ZnCuTi_Wal_t....uxd" file and "ZnCuTi_defocusing...uxd" ? 
Is there special or different set up to measure the defocusing data in XRD measurement ?

**Comment by kilir** [comment]
Hi,
one option for an analytical defocusing correction is the `'SchulzDefocusingKernel'` which produces a spherical function which you can then sample on the grid points of your pole figure for example.

If you want to measure it experimentally, you'd want to measure pole figures of an untextured powder for the 2theta angles of interest.

Hope that helps.

Cheers,
Rüdiger
